database/migrations/2021_01_21_175205_create_notification_reads_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateNotificationReadsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notification_reads', function (Blueprint $table) {
```

Comment:
Point for discussion - do we call this `notification_user` to match Laravel conventions?

Reply:
2d80f56